hs_core/models.py

```diff
@@ -3543,9 +3538,7 @@ def new_get_content_model(self):
 class CoreMetaData(models.Model):
     """Define CoreMetaData model."""
 
-    XML_HEADER = '''<?xml version="1.0"?>
-<!DOCTYPE rdf:RDF PUBLIC "-//DUBLIN CORE//DCMES DTD 2002/07/31//EN"
```

Comment:
Was this moved or do we no longer need this declaration?  This seems unrelated to this PR.